### PR TITLE
add some more comments on Cached Grains

### PR DESCRIPTION
**Pull request by mbgevers**
add some more comments on Cached Grains

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -277,9 +277,10 @@ saltgui_ipnumber_prefix:
 ```
 The display of the IP-numbers can simply be disabled by choosing a non-existing grain or by choosing a non-existing prefix.
 
+### Cached Grains
```
Comment (erwindon) [inline docs/README.md:280]: the rest of this chapter is not divided in 3rd-level sections. it either should be left out, or a section "### Regular Grains" should be added just below "## Grains"

Comment (erwindon) [inline docs/README.md:280]: the same structure should be applied below "## Pillars"

File: docs/README.md [hunk 2]
```diff
@@ -277,9 +277,10 @@ saltgui_ipnumber_prefix:
...
 SaltGUI will retrieve cached grains information when variable `saltgui_use_cache_for_grains` is set to `true`.
 In that case, unreachable minions will appear without warnings for that.
-In all cases, the information may be less accurate.
+In all cases, the information may be less accurate, however the Information may help to create a basic inventory, as by nature an Inventory is never absolutely up-to-date till its next inventory update run.
```
Comment (erwindon) [inline docs/README.md:283]: Information and Inventory should be without capital, do you have a German language background? :-)

Comment (erwindon) [inline docs/README.md:283]: should this explanation also be added to the Pillars section?

Comment (erwindon) [inline docs/README.md:283]: ", however the"
better to actually start a new sentence.
therefore:
". However, the"

Comment (erwindon) [inline docs/README.md:283]: the whole explanation is a bit unclear for me.
SaltGUI does not contribute to building this inventory as that inventory was already maintained by the salt-master. This change just uses that information.

**Review by erwindon — CHANGES_REQUESTED** [review]

**Comment by erwindon** [comment]
@mbgevers 
github closed this PR automatically because the base branch no longer exists.
feel free to re-open it... (after rebasing it to `main`)

File: docs/README.md (hunk 2 above)
Comment (mbgevers) [reply docs/README.md:283]: Hi,
lol yes i have, but actually Swiss not German.

Comment (mbgevers) [reply docs/README.md:283]: I have to read some more about Pillars, as i don't understand them very well yet, then of course i may add that.

Comment (mbgevers) [reply docs/README.md:283]: Will work on improvement here

Comment (mbgevers) [reply docs/README.md:283]: Sure, has to be improved

**Pull request closed without merging**